tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/ModifiableRecordSession.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Azure.Sdk.Tools.TestProxy.Common
+{
+    public class ModifiableRecordSession
+    {
+        public RecordMatcher CustomMatcher = null;
+
+        public RecordSession Session;
```

Comment:
```suggestion
        public RecordMatcher CustomMatcher { get; set;}

        public RecordSession Session { get; }
```

NIT: Convert to auto-properties.  I assume `Session` should be get-only.